### ISSUE TITLE
tzinfo behaviour differs between stdlib and backports

### ISSUE THREAD

**Issue by rikroe** [adamchainz/time-machine#261]
### Python Version

3.7, 3.8

### Package Version

2.7.0

### Description

Hi,

I noticed that the behaviour between `tzinfo` from `zoneinfo` and `backports.zoneinfo` (`backports.zoneinfo==0.2.1` on Python 3.7 & 3.8) differs, leading to my tests breaking if I want to check if a given timezone is actually set.

Example code:
```python
import datetime, sys
try:
    import zoneinfo
except ImportError:
    from backports import zoneinfo

import time_machine


@time_machine.travel(
    datetime.datetime(2011, 11, 28, tzinfo=zoneinfo.ZoneInfo("America/Los_Angeles")),
    tick=False,
)
def test_timezone():
    print(sys.version_info)
    dt = datetime.datetime.now()
    tz = dt.astimezone().tzinfo
    print(f"dt: {dt}")
    print(f"tz: {tz} - offset: {round(tz.utcoffset(datetime.datetime.now()).seconds / 60, 0)}")

test_timezone()
```

Output on Python 3.9 (as expected):
```
sys.version_info(major=3, minor=9, micro=9, releaselevel='final', serial=0)
dt: 2011-11-28 00:00:00
tz: PST - offset: 960.0
```

Output on Python 3.8 (not expected):
```
sys.version_info(major=3, minor=8, micro=12, releaselevel='final', serial=0)
dt: 2011-11-28 09:00:00
tz: CET - offset: 60.0
```

Thanks for having a look into this issue!

**Comment by adamchainz** [comment]
Fixed in 2.7.1, thanks for the clear bug report! The attempt to import from the backport package was missing 😅

**Comment by rikroe** [comment]
Many thanks for the quick fix!
